doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -181,17 +181,21 @@ To import the appliance:
 ]
 ----
 +
-. Use AWS-CLI tools to import the image. Run the following command:
+. Use AWS-CLI tools to import the disk as a snapshot by following the steps in the
+link://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-import-snapshot.html[Importing a Disk as a Snapshot Using VM Import/Export] documentation.
 +
 ----
-$ aws ec2 import-image --disk-containers file://containers.json --region BUCKET_REGION
+$ aws ec2 import-snapshot
 ----
 +
-. Check the progress of your imported image by running the following command:
+. Create the AMI from the snapshot by following the steps in the
+link://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html[Creating a Linux AMI from a Snapshot] documentation.
```

Comment:
Same as above. Please implement link to aws docs and direct users to seek out the topic.